$NetBSD: patch-doc_CMakeLists.txt,v 1.1 2022/12/09 12:08:56 gdt Exp $

Find where pkgsrc installs asciidoctor, rather than the normal place.

Perhaps a patch should be pushed upstream that looks for versioned
asciidoctor in addition to uadorned asciidoctor, or perhaps our cmake
should be taught about versioned programs, or perhaps something else.

--- doc/CMakeLists.txt.orig	2022-11-21 18:53:32.000000000 +0000
+++ doc/CMakeLists.txt
@@ -1,4 +1,4 @@
-find_program(ASCIIDOCTOR_EXE asciidoctor)
+find_program(ASCIIDOCTOR_EXE asciidoctor31)
 mark_as_advanced(ASCIIDOCTOR_EXE) # Don't show in CMake UIs
 
 if(NOT ASCIIDOCTOR_EXE)
